tests/math/math.cpp

```diff
@@ -18,36 +18,33 @@ limitations under the License.
 
 #include <libcellml>
 
+static const std::string emptyMath = "<math xmlns=\"http://www.w3.org/1998/Math/MathML\"/>\n";
```

Comment:
Shouldn't this variable be ALL CAPS?

Reply:
I don't mind either way. It's just that I have seen something similar in `tests/generator/generator.cpp`.

Reply:
Don't look in the tests directory for code conformance!

Reply:
Ok, I am going to replace `emptyMath` with `EMPTY_MATH`. No problem.